pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -189,6 +190,10 @@ public BrokerService(PulsarService pulsar) throws Exception {
 
         this.multiLayerTopicsMap = new ConcurrentOpenHashMap<>();
         this.pulsarStats = new PulsarStats(pulsar);
+        // register listener to capture zk-latency
+        ClientCnxnAspect.addListner((eventType, latencyMs) -> {
+            this.pulsarStats.recordZkLatencyTimeValue(eventType, latencyMs);
```

Comment:
We should also record fractions of Millis, since the `getData()` operations are going to be in the order of 0.1 to 0.5 millis most of the time